sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -113,13 +113,10 @@ WITH base AS (
   FROM
     `{{ events_view }}`
   WHERE
-    {% raw %}
-    {% if is_init() %}
```

Comment:
Removed the is_init to clean this up a bit.  Any new events stream table now should be created around the same time as the upstream events table anyway